src/test/java/edu/ucsb/cs156/happiercows/services/SystemInfoServiceImplTests.java

```diff
@@ -29,7 +32,24 @@ void test_getSystemInfo() {
     SystemInfo si = systemInfoService.getSystemInfo();
     assertFalse(si.getSpringH2ConsoleEnabled());
     assertFalse(si.getShowSwaggerUILink());
-    assertEquals("https://github.com/ucsb-cs156/proj-happycows", si.getSourceRepo());
+    assertEquals("https://github.com/ucsb-cs156-s24/proj-happycows-s24-4pm-5", si.getSourceRepo());
```

Comment:
```suggestion
    assertEquals("https://github.com/ucsb-cs156/proj-happycows", si.getSourceRepo());
```

probably, once you make the other changes